src/routes/(app)/import/+page.svelte

```diff
@@ -422,7 +423,7 @@
 
         // Define the main general status of the movie/show
         // In Ryot, it can be marked as multiple of the following, so choose the most relevant
-        const statusRanks = [
+        const statusRanks: [string, WatchedStatus][] = [
```

Comment:
Gave this a "tuple array" type so that it knows the second value is a `WatchedStatus`, fixed a type error below where we add `status` to the `ImportedList` object.